src/bare_etiss_processor/get_metrics.py

```diff
@@ -14,9 +14,16 @@
 Script to gather metrics on ROM,RAM,Stack and Heap usage.
 
 To produce the memory trace:
-- Compile etissvp_lib with SC_MEM_WRITE_TRACE defined (simple: uncomment at top of mem.cpp)
-- Invoke the run.sh script with the "nodmi" option
-- A file "pulpino_soc.dmem_memtrace.csv" should have been created
+- Enable tracing:
+  - Via INI file:
+  ```ini
+  [BoolConfigurations]
+    simple_mem_system.print_dbus_access=true
+    simple_mem_system.print_to_file=true
+  ```
+  - Via run_helper.sh script: `run_helper.sh ... trace`
+  - Via command line: `bare_etiss_processor ... --simple_mem_system.print_dbus_access=true --simple_mem_system.print_to_file=true`
+- A file "dBusAccess.csv" should have been created
```

Comment:
I dropped the SystemC-specific instructions in favor of standalone ETISS ones. If the SystemC memory tracing should also be covered here, let me know.